cfme/infrastructure/provider/__init__.py

```diff
@@ -314,9 +314,11 @@ def step(self):
 
     def resetter(self):
         # Reset view and selection
-        tb.select("Grid View")
-        sel.check(paginator.check_all())
-        sel.uncheck(paginator.check_all())
+        if tb.exists("Grid View"):
```

Comment:
I'm going to merge this but @mfalesni should we move the ```exists``` assertion inside the ```select```?